css-two-column-layout/styles.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.row {
+  display: flex;
+}
+
+.center-column {
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.column-one {
+  margin-left: 250px;
+  margin-right: 50px;
+}
+
+.column-two {
+  margin-right: 250px;
+  margin-left: 50px;
+}
+
+h1 {
+  font-family: Karla Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 45px;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+}
+
+.heading-text {
+  padding: 0 200px;
+  margin: 0 230px;
+  text-align: center;
+}
+
+p {
+  font-family: Karla, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #606060;
+  line-height: 27px;
+}
+
+h3 {
+  font-family: Karla, Arial, Helvetica, sans-serif;
+  font-weight: 600;
+}
+
+.img-and-text {
+  display: flex;
+}
+
+.column-third-left {
+  width: calc(100% / 3);
+}
+
+.column-third-right {
+  width: calc(100% / 3);
+}
+
+.column-two-thirds-left {
+  width: calc(200% / 3);
+  padding-right: 10px;
+  padding-top: 10px;
+}
+
+.column-two-thirds-right {
+  width: calc(200% / 3);
+  padding-left: 10px;
+  padding-top: 10px;
+}
```

Comment:
You should reduce this to just one `column-third` and one `column-two-thirds` class, there's no need for this repetition.